Dashboard/DashboardApp/Application/Components/CDP/CDPEngine.xml

```diff
@@ -3,13 +3,13 @@
 <!-- Default Engine CDP system component. -->
 <Component Name="CDPEngine" Model="CDPEngine" Description='Handles CDP process scheduling.' Activate="1">
```

Comment:
Don't really have to commit all files in CDP. Studio will generate them automatically if there are no changes from Model. You can look our example projects to see which files to commit.

Reply:
Removed CDP internal components that will be generated automatically.